src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -195,18 +195,41 @@ export const CustomRichBlocks = ({
                     </BlogLink>
                   );
                 } else {
-                  return (
-                    <BlogParagraph
-                      italic={el.props.italic}
-                      strikethrough={el.props.strikethrough}
-                      underline={el.props.underline}
-                      bold={el.props.bold}
-                      key={`blog-paragraph-${index}`}
-                    >
-                      {el.props.text}
-                    </BlogParagraph>
-                  );
+                  if (el.props.text.indexOf('\n') === -1) {
```

Comment:
Not really fan of this, what is the purpose of it exactly? you can use `nl2br` to do this

Reply:
Thanks, Sebastien! This condition should be met if there is no line-break within the strapi-paragraph.
In such cases we just return our default <BlogParagraph>. Compared to, if there is a "\n" found, it is split into an array and returns multiple <BlogParagraph>-components.

Feel free to share some more info on how to apply nl2br.

Reply:
something like this https://github.com/yosuke-furukawa/react-nl2br

Reply:
Thanks, good idea! Added it now